File: app/src/main/java/gateway/hitrontech/com/encryption/fragment/file_encryption_decryption/File/FileImpl.java
package gateway.hitrontech.com.encryption.fragment.file_encryption_decryption.File;

import gateway.hitrontech.com.encryption.bean.EncryptionBean;
import java.io.IOException;
import java.util.ArrayList;

public interface FileImpl {

  void readFile(ArrayList<EncryptionBean> list, String filePath) throws IOException;

  void writeFile(ArrayList<EncryptionBean> list, String filePath);

}

File: app/src/main/java/gateway/hitrontech/com/encryption/fragment/file_encryption_decryption/File/FileImpl.kt
package gateway.hitrontech.com.encryption.fragment.file_encryption_decryption.File

import gateway.hitrontech.com.encryption.bean.EncryptionBean
import java.util.*

interface FileImpl {

    fun readFile(list: ArrayList<EncryptionBean>, filePath: String)

    fun writeFile(list: ArrayList<EncryptionBean>, filePath: String)

}
